src/Symfony/Component/Console/Formatter/OutputFormatter.php

```diff
@@ -98,6 +98,19 @@ public function setStyle($name, OutputFormatterStyleInterface $style)
     }
 
     /**
+     * Adds a new style.
+     *
+     * @param string      $name       The style name
+     * @param string|null $foreground The style foreground color name
+     * @param string|null $background The style background color name
+     * @param array       $options    The style options
+     */
+    public function addStyle($name, $foreground = null, $background = null, array $options = array())
```

Comment:
I agree with you, but in that case it may feat because it does create an instance, it just adds it to a property instead or returning it, but what about adding a new line after to actually return it ;)